Bukkit/src/main/java/com/plotsquared/bukkit/listeners/PlayerEvents.java

```diff
@@ -2491,10 +2491,18 @@ public boolean entityDamage(Entity damager, Entity victim) {
                     return false;
                 }
             } else if (victim instanceof Creature) { // victim is animal
-                if (plot != null && (plot.getFlag(Flags.ANIMAL_ATTACK, false) || plot.getFlag(Flags.PVE, false) || plot
-                        .isAdded(plotPlayer.getUUID()))) {
-                    return true;
+                if (plot != null || plot.isAdded(plotPlayer.getUUID())) {
+                  return true; 
+                 }
+
+                if (plot.getFlag(Flags.ANIMAL_ATTACK, false)) {
+                  return false; 
+                 }
+
+                if (plot.getFlag(Flags.PVE, false)) { 
```

Comment:
This could be merged with the if statement above (that returns true), as it also requires that the plot isn't null.